frontend/app/controllers/spree/addresses_controller.rb

```diff
@@ -10,7 +10,7 @@ def index
 
     def create
       @address = try_spree_current_user.addresses.build(address_params)
-      if @address.save
+      if create_service.call(user: try_spree_current_user, address_params: @address.attributes).success?
         flash[:notice] = I18n.t(:successfully_created, scope: :address_book)
         redirect_to spree.account_path
       else
```

Comment:
@KacperMekarski shouldn't we pass the result to the view? will it still display validation errors?

Reply:
@damianlegawiec views are using @address variable which did not change.
I checked it locally, it works fine :)
Same thing happens with #update.